python/nano/tutorial/notebook/inference/pytorch/pytorch_context_manager.ipynb

```diff
@@ -0,0 +1,322 @@
+    "    ])\n",
+    "\n",
+    "    val_transform = transforms.Compose([\n",
+    "        transforms.Resize((224, 224)),\n",
+    "        transforms.ToTensor(),\n",
+    "        transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])\n",
+    "    ])\n",
+    "\n",
+    "    if not Path(\"data\").exists():\n",
+    "        # download dataset\n",
+    "        download_and_extract_archive(url=DATA_URL, download_root=\"data\", remove_finished=True)\n",
+    "\n",
+    "    data_path = Path(\"data/cats_and_dogs_filtered\")\n",
+    "    train_dataset = ImageFolder(data_path.joinpath(\"train\"), transform=train_transform)\n",
+    "    val_dataset = ImageFolder(data_path.joinpath(\"validation\"), transform=val_transform)\n",
+    "\n",
+    "    indices = torch.randperm(len(val_dataset))\n",
+    "    val_dataset = Subset(val_dataset, indices=indices[:val_size])\n",
+    "\n",
+    "    train_dataloader = DataLoader(dataset=train_dataset, batch_size=8, shuffle=True)\n",
+    "    val_dataloader = DataLoader(dataset=val_dataset, batch_size=8, shuffle=False)\n",
+    "\n",
+    "    return train_dataset, val_dataset\n",
+    "\n",
+    "train_dataset, val_dataset = prepare_model_and_dataset(model, val_size=500)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# To obtain the latency of single sample, set batch_size=1\n",
+    "train_dataloader = DataLoader(train_dataset, batch_size=1)\n",
+    "val_dataloader = DataLoader(val_dataset)\n",
+    "\n",
+    "from bigdl.nano.pytorch import InferenceOptimizer\n",
+    "optimizer = InferenceOptimizer()\n",
+    "optimizer.optimize(model=model,\n",
+    "                   training_data=train_dataloader,\n",
+    "                   thread_num=4,\n",
+    "                   latency_sample_num=30)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "openvino_model = optimizer.get_model(\"openvino_fp32\")\n",
+    "input_sample = torch.rand(1, 3, 224, 224)\n",
+    "\n",
+    "with InferenceOptimizer.get_context(openvino_model):\n",
+    "    output = openvino_model(input_sample)\n",
+    "    assert torch.get_num_threads() == 4  # this line just to let you know Nano has provided thread control automatically : )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "accelerated_model, option = optimizer.get_best_model()\n",
+    "input_sample = torch.rand(1, 3, 224, 224)\n",
+    "\n",
+    "with InferenceOptimizer.get_context(accelerated_model):\n",
+    "    output = accelerated_model(input_sample)\n",
+    "    assert torch.get_num_threads() == 4  # this line just to let you know Nano has provided thread control automatically : )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 📝 **Advanced Usage**\n",
```

Comment:
Would it be better to add another section called “Advanced Usage" or something like "Multi-Model"? From my understanding, the contents "Here are some rules that how we solve conflict between multi context managers:..." are more suitable to put inside a note box.